_test/test_sqw/worker_4tests_local.m

```diff
@@ -56,13 +57,13 @@
 [fbMPI,intercomm] = JobExecutor.init_frameworks(control_struct);
 % initiate file-based framework to exchange messages between head node and
 % the pool of workers
-
 %--------------------------------------------------------------------------
 % step 1 the initialization has been completed providing the
 % communicator for exchange between control node and the cluster and
 % between the clusters nodes. The control node communicator knows the
 % folder for communications
 %--------------------------------------------------------------------------
+
```

Comment:
does this need to be one long script? Would it be easier to understand and test if the steps were extracted as helper functions?

Reply:
I was tempted by that and thinking about splitting, but apparently you will find that splitting it into subfunction gets the functions with very very long list of parameters. The script itself is relatively simple -- error checking and logging takes all space. Probably not yet worth splitting.

Reply:
I will think about it again....

Reply:
Changed horace_init to copy this file from Herbert.

Reply:
Probably not this time -- too complex changes for too few benifits.